src/components/forms/CloudDatabaseForm.js

```diff
@@ -0,0 +1,267 @@
+import React, { Component } from "react";
+import { NumberInput } from "@carbon/react";
+import { titleCase } from "lazy-z";
+import {
+  buildFormDefaultInputMethods,
+  buildFormFunctions,
+} from "../component-utils";
+import { isRangeInvalid } from "../../lib/iam-utils";
+import { IcseNameInput, IcseToggle } from "../Inputs";
+import { IcseFormGroup } from "../Utils";
+import { IcseSelect } from "../Dropdowns";
+import { databaseInputChange } from "../../lib/forms";
+import PropTypes from "prop-types";
+
+/**
+ * CloudDatabaseForm
+ * @param {Object} props
+ * @param {configDotJson} props.configDotJson config dot json
+ * @param {slz} props.slz slz state store
+ */
+class CloudDatabaseForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { ...this.props.data };
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleToggle = this.handleToggle.bind(this);
+    buildFormDefaultInputMethods(this);
+    buildFormFunctions(this);
+  }
+
+  /**
+   * handle input change
+   * @param {event} event event
+   */
+  handleInputChange(event) {
+    this.setState(databaseInputChange(this.state, event));
```

Comment:
when changing `service` or `plan`, set value to `kebabCase`

Reply:
this is being done in  the `databaseInputChange` function already @jvallexm